app/src/main/java/ciserver/App.java

```diff
@@ -3,11 +3,24 @@
  */
 package ciserver;
 
+/**
+ * This class is the initial class used to start the whole CI server program
+ */
 public class App {
+    /**
+     * Gets a greeting, in this case "Hello world!"
+     *
+     * @return String of the greeting to print
+     */
     public String getGreeting() {
         return "Hello World!";
     }
 
+    /**
+     * Print a greeting then launch the CI server
+     *
+     * @param args the arguments to the program, not used
```

Comment:
Minor nitpick, but I'd probably write `@param args the *command line* arguments to the program; not used` (without the `*`, they are just to emphasize changes), just to be as clear as possible.

Reply:
Yeah I agree with that. Will fix